### ISSUE TITLE
Complementary auto generated normals disappear on chests during break animations

### ISSUE THREAD

**Issue by TBlazeWarriorT** [IrisShaders/Iris#1389]
### What happened?

Auto Generated Normals disappears from chests while they are being broken, the highlights/shadows go away while the break overlay is there
Bug only happens on Iris, tested on OptiFine and it was fine

### Screenshots

Chest with the missing normals https://cdn.discordapp.com/attachments/744253058258829313/968618804651380836/unknown.png
F3 screenshot, on the chest work while not being broken https://cdn.discordapp.com/attachments/744253058258829313/968619498762567760/unknown.png

### Minecraft Version

1.18.2

### Iris Version

iris-mc1.18.2-1.2.2-build.32.jar

### Sodium Version

sodium-fabric-mc1.18.2-0.4.1+build.15.jar

### Operating System

Windows 10

### What is your GPU?

RTX 2060

### Java Version

Bundled (17.0.1)

**Comment by coderbot16** [comment]
Uh... Do block entities like chests even have auto generated normals to begin with? Do you have Enhanced Block Entities installed?

**Comment by TBlazeWarriorT** [comment]
> Uh... Do block entities like chests even have auto generated normals to begin with? Do you have Enhanced Block Entities installed?

Yes Emin added normals to chests a bunch before I reported that, it can be seen on the image included
I re-tested just now without EBE on 1.2.5 and 1.18.2. It still glitches.

**Comment by coderbot16** [comment]
This should have been fixed in v1.4.4.
